packages/api-bindings/src/lens/utils/CollectModuleSettings.ts

```diff
@@ -52,3 +55,219 @@ export function findCollectModuleSettings(
 ): CollectModuleSettings | null {
   return collectable.openActionModules?.find(isCollectModuleSettings) ?? null;
 }
+
+export type CollectFee = {
+  amount: Erc20Amount;
+  rate?: FiatAmount;
+};
+
+export type FreeCollectPolicy = {
+  type: CollectPolicyType.FREE_COLLECT;
+  collectNft: EvmAddress | null;
+  followerOnly: boolean;
+  collectLimit: string | null;
+  endsAt: string | null;
+  contract: gql.NetworkAddress;
+};
+
+export type PaidCollectPolicy = {
+  type: CollectPolicyType.PAID_COLLECT;
+  collectNft: EvmAddress | null;
+  recipient: EvmAddress;
+  referralFee: number;
+  followerOnly: boolean;
+  collectLimit: string | null;
+  endsAt: string | null;
+  contract: gql.NetworkAddress;
+  fee: CollectFee;
+};
+
+export type MultirecipientCollectPolicy = {
+  type: CollectPolicyType.MULTIRECIPIENT_COLLECT;
+  collectNft: EvmAddress | null;
+  recipients: gql.Recipient[];
+  referralFee: number;
+  followerOnly: boolean;
+  collectLimit: string | null;
+  endsAt: string | null;
+  contract: gql.NetworkAddress;
+  fee: CollectFee;
+};
+
+export type CollectPolicy = FreeCollectPolicy | PaidCollectPolicy | MultirecipientCollectPolicy;
+
+export enum CollectPolicyType {
+  FREE_COLLECT = 'FREE_COLLECT',
+  PAID_COLLECT = 'PAID_COLLECT',
+  MULTIRECIPIENT_COLLECT = 'MULTIRECIPIENT_COLLECT',
+}
```

Comment:
Should we use `OpenActionType` values?

Reply:
I simplified it so we don't need type at all